p4-14/v1.0.4/tex/p4.pt

```diff
@@ -646,11 +646,11 @@ valid bit for a metadata field, defining a separate metadata flag to
 represent that field's validity is a reasonable work around.
 \end{adjustwidth}
 
-Only valid packet header fields may result in a match (when a value is
-specified for exact or ternary matches against the field), although a
-match operation may explicitly check if a header instance (or field)
-is valid. Only valid packet headers are considered for deparsing (see
-Section~\ref{sec:deparsing}).  
+When an invalid packet header is matched in a table (i.e., when a value is
+specified for exact or ternary matches against the field), its value is
```

Comment:
I don't see how the phrase 'when a value is specified for exact or ternary matches against the field' is clarifying anything here.  In fact, it seems to be muddying the waters by leaving out 'lpm' match_kind's.  Adding 'lpm' wouldn't really clarify anything, because then what about 'range' and user-defined match_kinds?  It seems leaving out the parenthesized phrase entirely is better.